### DISCUSSION TITLE
2005/05/03/%E5%BE%AE%E8%BD%AF%E5%AF%B9%E4%B8%AD%E6%96%87%E6%9C%89%E6%AD%A7%E8%A7%86%E5%98%9B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#18] [Announcements]
# 2005/05/03/%E5%BE%AE%E8%BD%AF%E5%AF%B9%E4%B8%AD%E6%96%87%E6%9C%89%E6%AD%A7%E8%A7%86%E5%98%9B

我一直是在英文模式下浏览编辑 blog 的。

https://blog.qizhen.xyz/2005/05/03/%E5%BE%AE%E8%BD%AF%E5%AF%B9%E4%B8%AD%E6%96%87%E6%9C%89%E6%AD%A7%E8%A7%86%E5%98%9B

**Comment by ruanqizhen** [comment]
小＆U
2005年05月4日 20:27 编辑
这个早就看出来了~没话说

回复
徐徐
2005年05月3日 14:43 编辑
轮到我沙发了 :)

回复
婵
2005年05月3日 10:49 编辑
我当然同意跟你交换链接了，不过现在在旅途中，等假期一过，我们就会是邻居了！

回复
